.eslintrc.json

```diff
@@ -24,15 +24,18 @@
       "files": ["*.spec.*"],
       "extends": "@zendeskgarden/eslint-config/plugins/jest",
       "rules": {
-        "no-console": "off"
+        "no-console": "off",
+        "jsx-a11y/label-has-associated-control": "off",
+        "react/button-has-type": "off",
+        "react/jsx-key": "off"
```

Comment:
Any particular reasons why these rules are turned off?

Reply:
They weren't really turned off, per se. This limits their scope to just spec tests. Previously they were applied to both stories and spec tests.